operator/mapper/redis.go

```diff
@@ -0,0 +1,47 @@
+package mapper
+
+import (
+	"fmt"
+
+	"github.com/exoscale/egoscale/v2/oapi"
+	"github.com/go-logr/logr"
+	exoscalev1 "github.com/vshn/provider-exoscale/apis/exoscale/v1"
+	"k8s.io/utils/pointer"
+)
+
+func MapRedisObservation(log logr.Logger, instance oapi.DbaasServiceRedis) exoscalev1.RedisObservation {
+	notifications, err := toNotifications(instance.Notifications)
+	if err != nil {
+		// just ignore those if they fail to map
+		log.Error(err, "unable to map notifications", "notifications", instance.Notifications)
+	}
```

Comment:
`MapRedisParameters` is a bit different since not being able to parse settings seems crucial to me, while not being able to parse notifications not so much. The difference is that `MapRedisObservation` would still succeed with the rest mapped even if notifications is not working. 

But I can still return a specific error which we can just log.